src/styles/page.css

```diff
@@ -0,0 +1,23 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use `*` selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for the browser.